CHANGES.rst

```diff
@@ -28,13 +28,17 @@ Improvements
   by users submitting an abstract (:issue:`3138`)
 - Add support for boolean (yes/no) and freetext questions in abstract
   reviewing (:issue:`3175`)
+- Add session types to sessions; allow marking session types as poster
+  so that the sessions with poster session type are displayed differently
+  in the timetable
```

Comment:
poster sessions aren't a new feature, so I would change this a bit, maybe something like this:

```
- Add support for custom session types (:issue:`3189`)
- Move poster session flag from session settings to session type settings
```